templates/customernotification.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Alerts</title>
+</head>
+<body itemscope itemtype="http://schema.org/EmailMessage">
+    <p>
+        Hello!
+        <br><br>
+        This notification is for your OpenShift managed cluster running OpenShift Data Foundation.
+        <br><br>
+        <strong>
+        {{ range .Alerts.Firing }}
+            {{ range .Labels.SortedPairs }}
+                {{ if eq .Name "alertname" }}
+                        {{ if eq .Value "CephClusterCriticallyFull" }}
+                            Your storage cluster utilization has crossed 80% and will become read-only at 85% utilized! Please free up some space or if possible expand the storage cluster immediately to prevent any service access issues. It is common to also be alerted to OSD devices entering near-full or full states prior to this alert.                    
+                        {{ end }}
```

Comment:
Add `<br/>` after each message to cover for cases where there are multiple alerts, just in case

Reply:
Done